python_backend/api/v1/endpoints/rag.py

```
@@ -0,0 +1,214 @@
+    try:
+        from services.rag_service import RAGService
+        return RAGService()
+    except ImportError as e:
+        raise HTTPException(status_code=503, detail=f"RAG 서비스를 사용할 수 없습니다: {str(e)}")
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"RAG 서비스 초기화 실패: {str(e)}")
+
+@router.post("/rag/ingest", response_model=DocumentIngestResponse)
+async def ingest_documents(
+    request: DocumentIngestRequest,
+    rag_service = Depends(get_rag_service)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)